README.md

```diff
@@ -35,7 +35,7 @@ See the [Eth 2.0 Phase 2 Wiki](https://hackmd.io/UzysWse1Th240HELswKqVA?view) fo
 * [General test format](specs/test_formats/README.md)
 * [Merkle proof formats](specs/light_client/merkle_proofs.md)
 * [Light client syncing protocol](specs/light_client/sync_protocol.md)
-* [Beacon node API for validator](specs/validator/0_beacon-node-validator-api.md)
+* [Beacon node API for validator](https://github.com/ethereum/eth2.0-APIs/blob/master/apis/validator/beacon-node-validator-api.md)
```

Comment:
ahh it's actually redundant to L45: https://github.com/ethereum/eth2.0-specs/pull/1399/files#diff-04c6e90faac2675aa89e2176d2eec7d8L45

Maybe remove this line?

Reply:
Good point! Removed.